pynicotine/gtkgui/search.py

```diff
@@ -551,7 +552,7 @@ def update_filter_comboboxes(self):
                 presets = (">10MiB", "<10MiB", "<5MiB", "<1MiB", ">0")
 
             elif filter_id == "filtertype":
-                presets = ("flac|wav|ape|aiff|wv|cue", "mp3|m4a|aac|ogg|opus|wma", "!mp3")
+                presets = ("audio", "image", "video", "text", "archive", "audio|image|text", "video|text")
```

Comment:
A demonstration of the not `!` operator is needed, suggest `!executable`.

Reply:
I think I would be fine with this after all, when using the revised extension list in my other comment.